helmfile/elastisys/networkpolicy/values.yaml

```
@@ -0,0 +1,2 @@
+cert-manager:
+  enabled: false
```

I think this should also include `namespaces`
```suggestion
cert-manager:
  enabled: false
namespaces: []
```